examples/Simple/SimpleConsumer.AppHost/Stub.cs

```diff
@@ -0,0 +1,7 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="Stub.cs" company="Akka.NET Project">
+//      Copyright (C) 2025 - 2025 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Akka.Event;
```

Comment:
just use some MSBuild variables to not apply that global using to an Aspire project

Reply:
Done